test/forge/ERC4626Test.sol

```diff
@@ -18,7 +18,7 @@ contract ERC4626Test is IntegrationTest, IMorphoFlashLoanCallback {
     }
 
     function testDecimals() public {
-        assertEq(vault.decimals(), loanToken.decimals() + ConstantsLib.DECIMALS_OFFSET, "decimals");
```

Comment:
If `loanToken.decimals() > 18` then `vault.decimals()` will be equal to `loanToken.decimals()`, so this test fails for loan tokens with > 18 decimals.